pages/index.js

```diff
@@ -1,5 +1,110 @@
-import PetList from "@/components/PetList/PetList";
+import styled from "styled-components";
+import StyledLink from "@/components/StyledLink/StyledLink";
+import Image from "next/image";
 
-export default function HomePage({ petCollection }) {
-  return <PetList petCollection={petCollection} />;
+export const StyledStartPage = styled.section`
```

Comment:
Working with an article element would make more sense here.
A section usually contains a heading, since this is not the case here, I would prefer an article.

```suggestion
export const StyledStartPage = styled.article`
```